### PR TITLE
Prepare release 3.2.0

### PR DESCRIPTION
**Pull request by albig**
This updates the documentation and rise the extension version number to 3.2.0.

### CODE REVIEW

File: Documentation/Introduction/Index.rst [hunk 1]
```diff
@@ -31,18 +33,23 @@ depositary copies, manuscripts and letters, prints, newspapers and magazines.
 
 Download of current version is `avaiable on GitHub <https://github.com/kitodo/kitodo-presentation>`_.
```
Comment (beatrycze-volk) [inline Documentation/Introduction/Index.rst]:
```suggestion
Download of current version is `available on GitHub <https://github.com/kitodo/kitodo-presentation>`_.
```

File: Documentation/Introduction/Index.rst [hunk 2]
```diff
@@ -95,38 +103,38 @@ templates and are thus fully customizable.
...
+===========
 
-Some examples of Kitodo.Presentation in action on the `digital collection of SLUB Dresden <http://digital.slub-dresden.de/>`_:
+Some examples of Kitodo.Presentation in action on the `Digital Collection of SLUB Dresden <https://digital.slub-dresden.de/>`_:
 
-.. figure:: ../Images/Introduction/KitodoPresentaion-SLUB1.PNG
-   :width: 500px
+http://digital.slub-dresden.de/id457052678
+
+.. figure:: ../Images/Introduction/Digitale\ Sammlungen\ \ Colloquia\ Lutheri\ conscripta\ a\ quibusdam\ et\ alia\ quaedam\ addita\ sunt.\ Thesaurus\ theologiae\ -\ Mscr.Dresd.A.180.d.png
```
Comment (beatrycze-volk) [inline Documentation/Introduction/Index.rst:112]: The image links are broken. Should they be kept here?

Comment (albig) [reply Documentation/Introduction/Index.rst:112]: Maybe the links look broken, but they a properly escaped.

That's what the [renderer ](https://docs.typo3.org/m/typo3/docs-how-to-document/master/en-us/RenderingDocs/Quickstart.html#render-documenation-with-docker)makes out of it:

DlfDocumentationImage

Comment (beatrycze-volk) [reply Documentation/Introduction/Index.rst:112]: Ah ok, because in file view was there no displayed image.

**Review by beatrycze-volk — APPROVED** [review]

**Review by beatrycze-volk — APPROVED** [review]

**Review by ByteParty — APPROVED** [review]

**Pull request merged**